server/job-scheduler.js

```diff
@@ -55,33 +101,41 @@ class JobScheduler {
           ;`;
 
         let replacements = { jobName: jobName};
+        //getting jobs based on above query
         const jobs = await models.sequelize.query(query, {
           type: models.sequelize.QueryTypes.SELECT,
           replacements: replacements
         });
 
+        //Lopping through all the jobs
         for(const job of jobs) {
-          //submit the dependant job's wu and record the execution in job_execution table for the statusPoller to pick
-          let wuid = await hpccUtil.getJobWuidByName(job.clusterId, job.name);      
-          console.log(
-            `submitting dependant job ${job.name} ` +
-            `(WU: ${wuid}) to url ${job.clusterId}/WsWorkunits/WUResubmit.json?ver_=1.78`
-            );
-          let wuInfo = await hpccUtil.resubmitWU(job.clusterId, wuid);    
-          let jobExecutionData = {
-            name: job.name, 
-            clusterId: job.clusterId, 
-            dataflowId: job.dataflowId, 
-            applicationId: job.application_id, 
-            jobId: job.id, 
-            jobType: job.jobType == 'Script' ? SUBMIT_SCRIPT_JOB_FILE_NAME : SUBMIT_JOB_FILE_NAME,
-            sprayedFileScope: job.sprayedFileScope,
-            sprayFileName: job.sprayFileName,
-            sprayDropZone: job.sprayDropZone,
-            status: 'submitted'
+          if(job.jobType === "Manual"){
```

Comment:
I cant remember why this check is needed here. This should just add the job to the scheduler irrespective of its type (manual) and then submitJob.js will take care of the special logic based on job type. Wont that work?

Reply:
This is required because when a manual job is scheduled for the future (corn), it is handled differently for manual and other jobs